src/internal/markers.ts

```diff
@@ -84,14 +85,19 @@ export type ZipMarker = MarkerBase<MarkerType.ZIP> & {
   readonly sources: (Marker | null)[];
 };
 
+export type DebounceTimeMarker = MarkerBase<MarkerType.DEBOUNCE_TIME> & {
+  readonly source: Marker | null;
```

Comment:
```suggestion
  readonly sources: [Marker | null];
```
The marker system was designed with the idea that each marker should use the same fields to list its sources and dependencies.
`detectGlitches` looks for the `sources` and `dependencies` fields to traverse the tree, and our streams-to-ardoq exporter also uses those fields to create the references.